sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/environment/KeyVaultPropertySource.java

```diff
@@ -35,4 +35,9 @@ public Object getProperty(String name) {
         return operations.getProperty(name);
     }
 
+    @Override
+    public boolean containsProperty(String name) {
+        return (getProperty(name) != null);
```

Comment:
We could remove the outter "()", @chenrujun is this the way to fix the bug? Is the return value of `getPropertyNames()` correct now?

Reply:
> this the way to fix the bug?

Yes.

> Is the return value of getPropertyNames() correct now?

No. It's still possible that a key can get value by `getProperty()`, but the key does not belong to the value returned by `getPropertyNames()`, it's a problem that first introduced here: https://github.com/microsoft/azure-spring-boot/pull/550/commits/cd12e47a465798d9b8ec2b6202dddd488205c508